pylib/mozhginfo/mozhginfo/push.py

```diff
@@ -0,0 +1,62 @@
+class Push(object):
```

Comment:
Hmm, I really haven't found what's cpm mean, the result google give me is https://github.com/alkant/cpm.
BTW, do you mean __cmp__ for cpm? @armenzg :)